docs/guides/developer/docs/development-process.md

```diff
@@ -22,9 +22,10 @@ this repository are explained later in this guide.
 
 ### Jira and GitHub
```

Comment:
Should we remove Jira from the title of the section?

Reply:
Yep!